testsuite/exp

```diff
@@ -380,9 +379,10 @@ def run(
             shell,
             new_exp_git_branch,
             git_sha,
-            forge_runner_duration_secs,
-            forge_test_suite,
-            dry_run,
+            duration=forge_runner_duration_secs,
```

Comment:
If you're using kwargs here.... why not make them all kwargs?

Why not replace the args with an "options" struct